### PR TITLE
Tank limit

### PR DESCRIPTION
**Pull request by mjmayank**

### CODE REVIEW

File: src/Screens/Logs/Log.tsx [hunk 1]
```diff
@@ -314,12 +314,12 @@ const Log: FunctionComponent<LogProps> = ({ navigation, route }) => {
...
                         <View style={styles.airTankLabelContainer}>
                           <Text style={styles.airTankLabelText}>
-                            {t('AIR_TANK')} 1
+                            {t('AIR_TANK')}
                           </Text>
                         </View>
                         <View style={styles.airTankMeasurementContainer}>
                           <Text style={styles.capacityText}>
-                            {airLimit}{' '}
+                            {diveLog.review.start_air}{' '}
```
Comment (Nnanyielugo) [inline src/Screens/Logs/Log.tsx:322]: Can you include a screenshot of how this page looks now?

**Review by Nnanyielugo — CHANGES_REQUESTED** [review]

File: src/Screens/Logs/Log.tsx (hunk 1 above)
Comment (mjmayank) [reply src/Screens/Logs/Log.tsx:322]: image

Comment (mjmayank) [reply src/Screens/Logs/Log.tsx:322]: i decided to change the design, from how it was in the original design

Comment (Nnanyielugo) [reply src/Screens/Logs/Log.tsx:322]: Got it, thank you.

**Review by Nnanyielugo — APPROVED** [review]

**Pull request merged**